fg/apps/main/management/commands/create_database_bionode.py

```diff
@@ -21,7 +21,7 @@
 import shutil
 import sys
 
-from fg.settings import LAB_NAME,TRASH_NAME
+from fg.config import LAB_NAME,TRASH_NAME
 
```

Comment:
This will bug out - it should be an import from settings.

Reply:
My bad! Fixed.